lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -34,11 +40,16 @@
                 },
 
                 postRender: function(){
-                    this.enrollView = new CourseEnrollView({
-                        $el: this.$('.course-actions'),
-                        model: this.model,
-                        context: this.context
-                    });
+                    HtmlUtils.setHtml(
+                        this.$('.course-actions'),
+                        HtmlUtils.HTML(
+                            new CourseEnrollView({
+                                model: this.model,
+                                context: this.context,
+                                enrollModel: this.enrollModel
+                            }).$el
+                        )
+                    );
```

Comment:
It would be preferably to just instantiate the enroll view and pass it the DOM element that it should self-render into.